docker-compose.example.yml

```diff
@@ -0,0 +1,72 @@
+version: '2'
+services:
+  database:
+    image: mariadb
+    volumes:
+      - "/srv/pterodactyl/database:/var/lib/mysql"
+    environment:
+      ## Database settings
+      ## change if you want it to be more secure.
+      - "MYSQL_ROOT_PASSWORD=apassword"
+      - "MYSQL_DATABASE=pterodb"
+      - "MYSQL_USER=ptero"
+      - "MYSQL_PASSWORD=pterodbpass"
+
+  cache:
+    image: redis:alpine
+
+  panel:
+    image: quay.io/pterodactyl/panel:latest
+    ports:
+      - "80:80"
+      - "443:443"
+    links:
+      - database
+      - cache
+    volumes:
+      - "/srv/pterodactyl/var/:/app/var/"
+    environment:
+      ## These are defaults and should be left alone
+      - "APP_ENV=production"
+      - "APP_DEBUG=false"
+      - "APP_THEME=pterodactyl"
+      - "APP_CLEAR_TASKLOG=720"
+      - "APP_DELETE_MINUTES=10"
+      - "APP_ENVIRONMENT_ONLY=false"
+      - "QUEUE_HIGH=high"
+      - "QUEUE_STANDARD=standard"
+      - "QUEUE_LOW=low"
+      ## Cache settings
+      - "CACHE_DRIVER=redis"
+      - "SESSION_DRIVER=redis"
+      - "QUEUE_DRIVER=redis"
+      - "REDIS_HOST=cache"
+      - "REDIS_PASSWORD=null"
+      - "REDIS_PORT=6379"
+      ## Domain settings
+      - "APP_URL=https://your.domain.here"
+      ## Timezone settings
+      - "APP_TIMEZONE=America/New_York"
+      ## Service egg settings
+      - "APP_SERVICE_AUTHOR=noreply@your.domain.here"
+      ## Database settings
+      ## change if you want it to be more secure.
+      - "DB_HOST=database"
+      - "DB_PORT=3306"
+      - "DB_DATABASE=pterodb"
+      - "DB_USERNAME=ptero"
+      - "DB_PASSWORD=pterodbpass"
+      ## Email settings
+      - "MAIL_FROM=noreply@your.domain.here"
+      - "MAIL_DRIVER=smtp"
+      - "MAIL_HOST=mail"
+      - "MAIL_PORT=1025"
+      - "MAIL_USERNAME=''"
+      - "MAIL_PASSWORD=''"
+      - "MAIL_ENCRYPTION=true"
+
+networks:
```

Comment:
do we need that in the example?

Reply:
I like to have options there even if not being used. It could be commented out instead.